requirements.txt

```diff
@@ -0,0 +1,19 @@
+docutils==0.15.2
+matplotlib==1.5.1
+Jinja2==2.8
```

Comment:
This is only needed for building the binaries / nightlies

Reply:
Removed.